plugin/utility/src/main/java/org/opencds/cqf/ruler/plugin/utility/ResolutionUtilities.java

```diff
@@ -470,42 +470,44 @@
 
 	/**
 	 * Returns the Resource at the specified location.
-	 * @param <ResourceType>    an IBaseResource type
-	 * @param theResourceDao    the DaoRegistry to use for resolution
-	 * @param theLocation       the location of the Resource to resolve
-	 * @param theFhirContext	the FhirContext to use to parse the resource
+	 * 
+	 * @param <ResourceType> an IBaseResource type
+	 * @param theResourceDao the DaoRegistry to use for resolution
+	 * @param theLocation    the location of the Resource to resolve
+	 * @param theFhirContext the FhirContext to use to parse the resource
 	 * @return the Resource at the specified location
 	 */
 	@SuppressWarnings("unchecked")
 	public default <ResourceType extends IBaseResource> ResourceType resolveByLocation(
-		DaoRegistry theResourceDao, String theLocation, FhirContext theFhirContext)
-		throws IOException {
-	  String json = stringFromResource(theLocation);
-	  IBaseResource resource = theFhirContext.newJsonParser().parseResource(json);
-	  IFhirResourceDao<IBaseResource> dao = theResourceDao.getResourceDao(resource.getIdElement().getResourceType());
-	  if (dao == null) {
-		return null;
-	  } else {
-		dao.update(resource);
-		return (ResourceType) resource;
-	  }
+			DaoRegistry theResourceDao, String theLocation, FhirContext theFhirContext)
```

Comment:
Whoops. Meant to put this into a test utilities class.

Reply:
I went back and forth whether to put this in a test utilities or in this utility.  I landed on "maybe a server would want to load from a location?"

Reply:
Moved to a test-utility plugin